com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/ActivitiExceptionEventHandler.java

```diff
@@ -28,34 +30,44 @@ public ActivitiExceptionEventHandler(ProgressMessageService progressMessageServi
         this.progressMessageService = progressMessageService;
     }
 
-    public void handle(ActivitiEvent event) {
-        if (!(event instanceof ActivitiExceptionEvent)) {
+    public void handle(FlowableEvent event) {
+        if (!(event instanceof FlowableExceptionEvent) && !(event instanceof FlowableEngineEvent)) {
             return;
         }
 
-        ActivitiExceptionEvent activitiExceptionEvent = (ActivitiExceptionEvent) event;
-        String activitiExceptionStackTrace = ExceptionUtils.getStackTrace(activitiExceptionEvent.getCause());
-        LOGGER.error(activitiExceptionStackTrace);
-        String activitiExceptionMessage = activitiExceptionEvent.getCause()
+        FlowableExceptionEvent flowableEngineEvent = (FlowableExceptionEvent) event;

```

Comment:
Why is the variable called flowableEngineEvent when you're casting to FlowableException event?